config/prow/cluster/monitoring/mixins/grafana_dashboards/slo.jsonnet

```diff
@@ -0,0 +1,45 @@
+local config =  import 'config.libsonnet';
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+
+
+local dashboardConfig = {
+        uid: config._config.grafanaDashboardIDs['slo.json'],
+    };
+
+dashboard.new(
+        'SLO Compliance Dashboard',
+        time_from='now-7d',
+        schemaVersion=18,
+      )
+.addPanel(
+    graphPanel.new(
+        'Prow overall SLO compliance',
+        description='slo_prow_ok',
+        datasource='prometheus',
+        legend_rightSide=true,
+    )
+    .addTarget(prometheus.target(
+        'slo_prow_ok'
+    )), gridPos={
```

Comment:
Not sure if this was auto formatted but I normally see the Pos params on one line.

Reply:
The jsonnet files are all manually formatted. I reformatted the `gridPos` parameter to a single line.